example/sample_data/sample_table_owner.csv

```diff
@@ -0,0 +1,3 @@
+database,cluster,schema_name,table_name,owners
+hive,gold,test_schema,test_table1,"roald.amundsen@example.org,chrisc@example.org"
+dynamo,gold,test_schema,test_table2,roald.amundsen@example.org
```

Comment:
I think it’s pretty important in sample data to highlight that this does not have to be populated up front - but you can “fill in the blanks” along the way, as owners emerge. So: 

```suggestion
dynamo,gold,test_schema,test_table2,
```

Background Slack thread: https://amundsenworkspace.slack.com/archives/CGFBVT23V/p1569872497220000?thread_ts=1569865635.187900&channel=CGFBVT23V&message_ts=1569872497.220000